File: plugin_loader/src/server_api/sc/protocol/requests/CancelRequest.java
package sc.protocol.requests;

import com.thoughtworks.xstream.annotations.XStreamAlias;
import com.thoughtworks.xstream.annotations.XStreamAsAttribute;

@XStreamAlias("cancel")
public class CancelRequest implements ILobbyRequest
{
	@XStreamAsAttribute
	public String	roomId;

	public CancelRequest(String roomId)
	{
		this.roomId = roomId;
	}
}

File: sdk/src/server-api/sc/protocol/requests/CancelRequest.kt
package sc.protocol.requests

import com.thoughtworks.xstream.annotations.XStreamAlias
import com.thoughtworks.xstream.annotations.XStreamAsAttribute

/** Used to cancel game and delete a GameRoom. */
@XStreamAlias("cancel")
data class CancelRequest(
        @XStreamAsAttribute
        val roomId: String
): AdminLobbyRequest
